device/mellanox/x86_64-mlnx_msn4800-r0/sensors.conf

```diff
@@ -0,0 +1,189 @@
+################################################################################
```

Comment:
can we have it empty?

Reply:
i don't understand, what do you mean empty? empty file?

Reply:
I dont think the sensor file is accurate. if it is good. if not, check if empty file or missing file will still work. better to have the minimum required for now

Reply:
the sensor file was taken from a system that is similar to 4800 (according to Vadim.P). we can take it and maybe change it later when the system will be available.

running on empty file is causing errors on log